ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -932,13 +932,16 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
   }
 
   private func profileName(withFont font: Font) -> String? {
-    return font.source.first { !($0.contains(".mobileconfig")) }
+    return font.source.first { $0.lowercased().hasSuffix(FileExtensions.configurationProfile) }
```

Comment:
Not using `hasExtension(_:)` since I think it's better for that extension to be internal in KeymanEngine.